src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -81,6 +81,8 @@
     private Handler<AsyncResult<Void>> subscribeHandler;
     // handler called after an unsubscription request
     private Handler<AsyncResult<Void>> unsubscribeHandler;
+    // handler called after an get subscriptions request
+    private Handler<AsyncResult<Set<TopicPartition>>> listSubscriptionsHandler;
```

Comment:
In this case I don't see the need for having a field for the handler. The listSubscriptions method can get the handler as parameter as it happens for other methods. Wdyt?